INSTALL.md

```diff
@@ -75,6 +75,51 @@ $ curl -sSL https://takeout.waffle.io/get | sudo bash
 
 ![Step 8](doc/screenshots/6.png)
 
+### Airgap Installation
+
+For installations without external internet access or are behind a proxy, you can install Waffle Takeout using the following steps.
```

Comment:
"or are" sounds a little awkward. Maybe "For installations who don't have easy access to the internet"